apps/interpreter/src/run-action.ts

```diff
@@ -35,9 +34,9 @@ export async function runAction(
     return process.exit(ExitCode.FAILURE);
   }
 
-  const currentDir = getCurrentDir();
+  const currentDir = process.cwd();
   const workingDir = currentDir;
-  const absoluteFilePath = path.join(currentDir, filePath);
+  const filePathRelativeToCurrentDir = path.relative(currentDir, filePath);
```

Comment:
Does this not introduce some weirdness again that makes calling a jv file dependent on where the interpreter is running?

Reply:
> Not sure what the effects are of the directory logic changing in the run-action.ts, if you're confident in that change I'd trust your knowledge though.

Ah yes.. I noticed with our current version we stopped supporting absolute paths :(
The new directory logic enables absolute paths again, by mapping them to a relative path from the current dir.
I'm open to better solutions if you have some in mind :)

Reply:
I think at all times, all paths should be relative to the model location without exception :).

Reply:
Ah I mean for the interpreter, sorry!
So `jv /home/me/model.jv` wasn't working.
Import paths in models are always relative.

Reply:
Ah, yeah, then I love it.